packages/select/src/components/ui/GroupedMultiSelect.tsx

```diff
@@ -65,8 +65,15 @@ const resolveIconColor = (
   isFocused: boolean
 ): string | undefined =>
   isFocused
-    ? theme.menu.selectedItemHoverTextColor
-    : theme.menu.selectedItemTextColor;
+    ? theme.menu.selectedItemHoverIconColor
+    : theme.menu.selectedItemIconColor;
+
+const resolveSelectedTextColor = (
+  theme: SelectTheme,
+  isSelected: boolean
+): string => {
+  return isSelected ? theme.menu.selectedItemTextColor : "";
```

Comment:
See other comment, but in general, return undefined instead of empty string.

Reply:
Fixed!